pkg/controller/nodeset/node/controller.go

```diff
@@ -251,35 +299,46 @@ func (c *Controller) syncHandler(key string) error {
 		return nil
 	}
 
-	glog.Infof("NodeSet seen %q", nodeset.Name)
+	switch true {
```

Comment:
switch true?

Reply:
I intended to have multiple checks on .spec.* == status.*
Turns out, that does not work

Reply:
of course it does:
```
switch {
case a == 42: ...
case b == 20 || a = 10: ...
}

Reply:
I was not aware that you can simplify this in go. Really cool!
But the idea does not fit well for the structure im going with